README.md

```diff
@@ -33,7 +33,7 @@ Kopia supports saving your [encrypted](https://kopia.io/docs/features/#end-to-en
 * Google Cloud Storage
 * Any remote server or cloud storage that supports WebDAV
 * Any remote server or cloud storage that supports SFTP
-* Dropbox, OneDrive, Google Drive, and all cloud storage supported by Rclone
+* The cloud storages supported by Rclone
```

Comment:
I would specifically mention that not all providers are supported. Some for example don't guarantee unique filenames, don't have server-assigned timestamps or have no basic consistency.